README.md

```diff
@@ -1148,7 +1148,7 @@
     var OBJEcttsssss = {};
     var this_is_my_object = {};
     function c() {}
-    var u = new user({
+    var u = new User({
```

Comment:
u is bad, not the fact that "user" should be "User"

The constructor rule is shown in the next example below (PascalCase)